.appveyor.yml

```diff
@@ -6,10 +6,10 @@ install:
   - python --version
   - pip install lxml
   - pip install pypiwin32
-  - choco install dmd
-  - choco install ldc
-  - choco install swig
-  - choco install vswhere
+  - choco install --allow-empty-checksums dmd
```

Comment:
what's the --allow-empty-checksums resolving? Were they not installing previously?

Reply:
I was getting failures to install sometimes due to empty checksums, but it seems intermittant and maybe related to connections?

```
choco install swig
Chocolatey v0.10.8
Installing the following packages:
swig
By installing you accept licenses for the packages.
Progress: Downloading swig 3.0.9... 100%
swig v3.0.9 [Approved]
swig package files install completed. Performing other installation steps.
Attempt to get headers for http://downloads.sourceforge.net/project/swig/swigwin/swigwin-3.0.9/swigwin-3.0.9.zip failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'http://downloads.sourceforge.net/project/swig/swigwin/swigwin-3.0.9/swigwin-3.0.9.zip'. Exception calling "GetResponse" with "0" argument(s): "The operation has timed out"
Downloading swig 
  from 'http://downloads.sourceforge.net/project/swig/swigwin/swigwin-3.0.9/swigwin-3.0.9.zip'
Progress: 100% - Completed download of C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\swig\3.0.9\swigwin-3.0.9.zip (10.29 MB).
Download of swigwin-3.0.9.zip (10.29 MB) completed.
WARNING: Missing package checksums are not allowed (by default for HTTP/FTP, 
 HTTPS when feature 'allowEmptyChecksumsSecure' is disabled) for 
 safety and security reasons. Although we strongly advise against it, 
 if you need this functionality, please set the feature 
 'allowEmptyChecksums' ('choco feature enable -n 
 allowEmptyChecksums') 
 or pass in the option '--allow-empty-checksums'. You can also pass 
 checksums at runtime (recommended). See choco install -? for details.
The integrity of the file 'swigwin-3.0.9.zip' from 'http://downloads.sourceforge.net/project/swig/swigwin/swigwin-3.0.9/swigwin-3.0.9.zip' has not been verified by a checksum in the package scripts.
Do you wish to allow the install to continue (not recommended)?
[Y] Yes [N] No (default is "N")
  Confirmation (`-y`) is set.
  Respond within 30 seconds or the default selection will be chosen.
ERROR: Empty checksums are no longer allowed by default for non-secure sources. Please ask the maintainer to add checksums to this package. In the meantime if you need this package to work correctly, please enable the feature allowEmptyChecksums, provide the runtime switch '--allow-empty-checksums', or pass in checksums at runtime (recommended - see 'choco install -?' / 'choco upgrade -?' for details). It is strongly advised against allowing empty checksums for non-internal HTTP/FTP sources.
The install of swig was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\swig\tools\chocolateyInstall.ps1'.
 See log for details.
Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures
 - swig (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\swig\tools\chocolateyInstall.ps1'.
 See log for details.
Did you know the proceeds of Pro (and some proceeds from other 
 licensed editions) go into bettering the community infrastructure?
 Your support ensures an active community, keeps Chocolatey tip top,
 plus it nets you some awesome features! 
 https://chocolatey.org/compare
Command exited with code -1
```